gs_bucket_backup/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# tag 258.0.0
```

Comment:
Nit: let's rename this container to `gke-gs-bucket-backup`. A few reasons:

- all other container names use hyphens rather than underscores
- we'll likely have a pile of GKE-related helper containers, so naming them with a prefix will aid in discovery
- in order to update the name, we'll have to touch the corresponding config references elsewhere, which will greatly aid in review here